src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/general.scss

```diff
@@ -2,14 +2,10 @@
     width: 95px;
 }
 
-#content-form {
+#content-form,
+.content-form {
```

Comment:
@danrot the `BC Break` is the css using the `#content-form` so custom bundles using currently the content-form for styling the highlight section will not look correctly when I remove`#content-form`.

For the classname I would prefer `.form` but don't know if this conflicts.